cmd/cluster-agent/subcommands/start/command.go

```diff
@@ -172,7 +173,7 @@ func start(log log.Component, config config.Component, telemetry telemetry.Compo
 	}
 
 	// Initialize remote configuration
-	var rcClient *remote.Client
+	var rcClient *client.Client
```

Comment:
Would be nice to rename import to `remoteclient`, because it's not super readable otherwise.